pkg/apis/extensions/v1beta1/types.go

```diff
@@ -1036,3 +1037,58 @@ type PodSecurityPolicyList struct {
 	// Items is a list of schema objects.
 	Items []PodSecurityPolicy `json:"items"`
 }
+
+// Template contains the inputs needed to produce a Config.
```

Comment:
Not sure.  What is the general process for getting an api review? [this doc](https://github.com/kubernetes/kubernetes/blob/7ab0a9f6345fd79a20073b38899dc9d9ddb62ff0/docs/devel/api_changes.md#so-you-want-to-change-the-api) suggests targeting the extensions group in the initial proposal